src/Controls/tests/DeviceTests/Elements/CheckBox/CheckBoxTests.cs

```diff
@@ -30,5 +43,31 @@ public async Task UpdatingCheckBoxBackgroundColorUpdatesBackground(string colorS
 
 			await ValidateHasColor<CheckBoxHandler>(checkBox, color);
 		}
+
+		[Fact(DisplayName = "Does Not Leak")]
+		public async Task DoesNotLeak()
```

Comment:
@jonathanpeppers does it make sense to standardize these tests? These are all basically the same format yea? Can we just use InlineData or a class to provide all the control types and just permutate through them all?

Reply:
I can do that after the current round of PRs are in.

It could probably be a parameterized test in `MemoryTests.cs`, as long as the test is reasonably searchable/runnable in the device tests menu.